### PR TITLE
Initial Seeding Code

### PR DESCRIPTION
**Pull request by kataing**
Completed:

[ Server ] 
- connects to database
- sends a single GET request to database to retrieve all product data
** Reasoning: Upon page load, the request will be triggered, and all search filtering will be done on the client-side **

[ Database ]
- defines Products schema
- lists all image urls (stored on s3) | File: image.js
- populates all product entries into a seedData array | File: seed.js
- seedData is then seeded into the database | File: seed.js

### CODE REVIEW

**Review by impromptuu — APPROVED** [review]
Looks good, make sure you make a gist showing your route works and post your link to the deliverables trackers

File: database/seed.js [hunk 1]
```diff
@@ -74,32 +80,34 @@ var brands = [
...
-var buildSeedData = (productTypes) => {
+const buildSeedData = (productTypes) => {
 
-  var makeItemEntries = (item, itemUrls) => {
-    for (var i = 0; i < itemUrls.length; i++) {
-      var entry = {};
-      // var nameExtension = '';
+  const makeItemEntries = (item, itemUrls) => {
+    for (let i = 0; i < itemUrls.length; i++) {
+      let entry = {};
```
Comment (kbsjsn) [inline database/seed.js:94]: Can you assign variables without reference type?

**Comment by kbsjsn** [comment]
Looks good. Please see question regarding reference type on variables in database/seed.js

**Comment by kataing** [comment]
@impromptuu @kbsjsn 
I've added the gist link for the sample API call below: 
https://gist.github.com/kataing/93a0394461db2c6042880a50b1a3e9f0

File: database/seed.js (hunk 1 above)
Comment (kataing) [reply database/seed.js:94]: Good catch! I updated my seed file, and sent an updated pull request.

**Pull request merged**